p/themes/base-theme/template.css

```diff
@@ -1180,7 +1180,7 @@ br {
 .nav label {
 	padding: 0;
 	display: none;
-	width: 10%;
+	width: 20%;
```

Comment:
I don't know if this `.nav label` selector occurs anywhere else, at a quick glance it looks like `70px` for example might make more sense? (Where the `px` as opposed to `%` is the salient point, not the exact value.)

Reply:
I do not understand why it makes more sense to switch from % to px.

Could not find another place where `.nav label` is used

Reply:
The question is, why `<label>` is used for prev/next.
IMHO it is not semantic correct.
(But: It should not be the point for this PR)

Reply:
> I do not understand why it makes more sense to switch from % to px.

Because 10% is clearly meant as 10% of 640px = 64px. That's when it looks good.

That's what's causing the problem. The reason 20% works is because 20% of 250px (the min-width) is 50px while 10% is only 25px. But then when you get to wider screens suddenly we're talking 128px instead of 64px. It doesn't look horrible but it definitely looks different.

Reply:
I know `git blame`

Reply:
I "just" doubled the previous setting (`10%` => `20%`).

I prefered the % because the font size of the arrows is also in %
https://github.com/FreshRSS/FreshRSS/blob/68dc18a7aa2d9594b0120d1754ab6f6340fc93f1/p/themes/base-theme/template.css#L1194

Reply:
Those are very different percentages. That's 0.9 * 1000% of what almost everyone leaves at the default font size of 16px, i.e., "always" 144px. This is actually a bit of a case of https://github.com/FreshRSS/FreshRSS/issues/448, albeit not one that's likely to ever become problematic.

The other value is as I said above, a percentage of the width.

To unify it so that they would scale together appropriately, that should be:

```css
font-size: 9rem; /* usually 144 px */
width: 4rem; /* usually 64px */
```

But whatever value rolled out of that (e.g., a user who sets it at 12px would get 108px and 48px), it'd always be in sync. A percentage of a font size and a percentage of a width won't do that. ;-)